[2024-09-29T12:46:04.420+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-29T12:46:04.542+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v04.postgres_to_s3 scheduled__2022-11-11T00:00:00+00:00 [queued]>
[2024-09-29T12:46:04.611+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v04.postgres_to_s3 scheduled__2022-11-11T00:00:00+00:00 [queued]>
[2024-09-29T12:46:04.630+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-29T12:46:04.731+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2022-11-11 00:00:00+00:00
[2024-09-29T12:46:04.810+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=23246) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-29T12:46:04.850+0000] {standard_task_runner.py:72} INFO - Started process 23267 to run task
[2024-09-29T12:46:04.843+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v04', 'postgres_to_s3', 'scheduled__2022-11-11T00:00:00+00:00', '--job-id', '1090', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmp_pyt5eu7']
[2024-09-29T12:46:04.869+0000] {standard_task_runner.py:105} INFO - Job 1090: Subtask postgres_to_s3
[2024-09-29T12:46:04.989+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:46:05.157+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_hooks_v04.postgres_to_s3 scheduled__2022-11-11T00:00:00+00:00 [running]> on host 6851fee35b72
[2024-09-29T12:46:05.165+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:46:05.662+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks_v04' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2022-11-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-11-11T00:00:00+00:00'
[2024-09-29T12:46:05.668+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-29T12:46:05.714+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/operator_helpers.py:175 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2024-09-29T12:46:05.755+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-29T12:46:05.819+0000] {dag_with_postgres_hooks.py:34} INFO - Saved orders data in text file: dags/get_orders_20221111.txt
[2024-09-29T12:46:05.907+0000] {base.py:84} INFO - Retrieving connection 'minio_conn'
[2024-09-29T12:46:05.925+0000] {connection_wrapper.py:391} INFO - AWS Connection (conn_id='minio_conn', conn_type='aws') credentials retrieved from extra.
[2024-09-29T12:46:05.931+0000] {logging_mixin.py:190} WARNING - <string>:9 AirflowProviderDeprecationWarning: extra['host'] is deprecated and will be removed in a future release. Please set extra['endpoint_url'] instead
[2024-09-29T12:46:08.144+0000] {dag_with_postgres_hooks.py:43} INFO - Orders file /tmp/tmptonn0j0y20221111 has been pushed to S3!
[2024-09-29T12:46:08.151+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-29T12:46:08.227+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-29T12:46:08.237+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v04, task_id=postgres_to_s3, run_id=scheduled__2022-11-11T00:00:00+00:00, execution_date=20221111T000000, start_date=20240929T124604, end_date=20240929T124608
[2024-09-29T12:46:08.353+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-29T12:46:08.456+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-29T12:46:08.464+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
